test/test_hmc_util.py

```diff
@@ -454,10 +458,10 @@ def model(data):
         5,  14,  13,  22,
     ])
 
-    rngs = random.split(random.PRNGKey(1), 12)
+    rngs = random.split(random.PRNGKey(1), 2)
```

Comment:
it is unnecessary to test the correctness of batched `rngs` with batch=12. I think 2 is enough.